redbeams/src/main/java/com/sequenceiq/redbeams/controller/v4/databaseserver/DatabaseServerV4Controller.java

```diff
@@ -76,10 +76,10 @@ public DatabaseServerAllocationOutcomeV4Response create(AllocateDatabaseServerV4
 
     @Override
     public DatabaseServerTerminationOutcomeV4Response terminate(TerminateDatabaseServerV4Request request) {
-        DatabaseServerConfig server = redbeamsTerminationService.terminateDatabaseServer(request.getName(), request.getEnvironmentId());
+        redbeamsTerminationService.terminateDatabaseServer(request.getName(), request.getEnvironmentId());
         // FIXME the converter for this doesn't exist yet, and probably needs to convert something
```

Comment:
Is this comment still needed?

Reply:
Sort of. Bill's status work will touch this though, so I'd vote to leave it and let Bill handle it.

Reply:
Yeah, I did write a converter for allocation responses under CB-1247, but not for termination responses here. That would be one of the very next things to add.